freshmaker/errata.py

```diff
@@ -514,3 +514,23 @@ def get_attached_build_nvrs(self, errata_id):
             for build in builds
             for nvr in build.keys()
         }
+
+    def get_build_rpm_nvrs(self, errata_id):
```

Comment:
Any reason of creating this new API? there is already `get_binary_rpm_nvrs` exist.

Reply:
Ah. I haven't noticed it. I was trying to add a new api like get_cve_affected_rpm_nvrs. Thanks for pointing this out. Fixed.